src/Button/Button.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import propTypes from 'prop-types'
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({ className, children, onClick, type, ref, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const classnames = require('classnames')
+  const levers = {}
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  return (
+    <button
+      className={classnames('su-button', levers.button, className)}
+      ref={ref}
+      onClick={onClick}
+      type={type}
+      {...props}
+    >
+      {children}
+    </button>
+  )
+}
+
+Button.propTypes = {
+  // HTML Button type.
+  type: propTypes.oneOf(['button', 'reset', 'submit']),
+
+  // Optional click handler
+  onClick: propTypes.func,
```

Comment:
Looks like we can attach an action to this:
https://storybook.js.org/docs/react/essentials/actions